resources/views/layouts/navigation.blade.php

```diff
@@ -12,26 +12,27 @@
 
                 <!-- Navigation Links -->
                 <div class="hidden space-x-8 sm:-my-px sm:ms-10 sm:flex">
-                    <x-nav-link :href="route('dashboard')" :active="request()->routeIs('dashboard')">
+                    <x-nav-link :href="route('dashboard')" :active="request()->routeIs('dashboard')" style="color:#26A7DE;">
                         {{ __('Dashboard') }}
                     </x-nav-link>
-		        <x-nav-link :href="route('chirps.index')" :active="request()->routeIs('chirps.index')">
+		        <x-nav-link :href="route('chirps.index')" :active="request()->routeIs('chirps.index')" style="color:#26A7DE;">
 	            	    {{ __('Chirps') }}
 	            </x-nav-link>
-                <x-nav-link :href="route('chirps.latest')" :active="request()->routeIs('chirps.latest')">
+                <x-nav-link :href="route('chirps.latest')" :active="request()->routeIs('chirps.latest')" style="color:#26A7DE;">
                         {{ __('Recent Chirps') }}
                 </x-nav-link>
-
+                <x-nav-link :href="route('user.index')" :active="request()->routeIs('user.index')" style="color:#26A7DE;">
+                    {{ __('All Chirpers') }}
+                </x-nav-link>
                 </div>
             </div>
 
             <!-- Settings Dropdown -->
             <div class="hidden sm:flex sm:items-center sm:ms-6">
                 <x-dropdown align="right" width="48">
                     <x-slot name="trigger">
-                        <button class="inline-flex items-center px-3 py-2 border border-transparent text-sm leading-4 font-medium rounded-md text-gray-500 bg-white hover:text-gray-700 focus:outline-none transition ease-in-out duration-150">
+                        <button class="inline-flex items-center px-3 py-2 border border-transparent text-sm leading-4 font-medium rounded-md text-gray-500 bg-white hover:text-gray-700 focus:outline-none transition ease-in-out duration-150" style="background-color:#26A7DE;color:white;">
                             <div>{{ Auth::user()->name }}</div>
```

Comment:
add null protection here in case there is no current logged in user

Reply:
I added logic to navigation.blade.php to change name to "guest" in the case that they can still access said page. Otherwise, all routes now have middleware auth, so if any page is manually entered, they just get redirected to the login page.